### PR TITLE
Add feedback footer

### PR DESCRIPTION
**Pull request by tjoyal**
Screen Shot 2022-05-01 at 6 39 13 PM

### CODE REVIEW

File: app/javascript/components/layouts/footer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { AppProvider, FooterHelp, Link } from '@shopify/polaris'
+
+class Footer extends React.Component {
+  render () {
+    return (
+      <AppProvider>
+        <FooterHelp>
+          We'd love to hear about your experience; you can{' '}
+          <Link url={ "https://docs.google.com/forms/d/e/1FAIpQLSdHStoRIefu86gEpKa1cSqakpv2m9x4SCifdO_4IymwgjN2fw/viewform?usp=pp_url&entry.1673643580=" + document.getElementById("shopify-app-init").dataset.shopOrigin } external={true}>
```
Comment (christianblais) [inline app/javascript/components/layouts/footer.js:10]: Just external (without the ={true}) should be enough. FYI.

File: app/javascript/components/layouts/footer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { AppProvider, FooterHelp, Link } from '@shopify/polaris'
+
+class Footer extends React.Component {
+  render () {
+    return (
+      <AppProvider>
```
Comment (christianblais) [inline app/javascript/components/layouts/footer.js:7]: Are we sure AppProvider is required here? I thought it was supposed to surround the entirety of the app, once.

File: app/views/layouts/embedded_app.html.erb [hunk 3]
```diff
@@ -32,6 +32,7 @@
 
             <%= yield %>
 
+            <%= react_component("layouts/footer", { }) %>
             <br><br>
```
Comment (christianblais) [inline app/views/layouts/embedded_app.html.erb:35]: Now that we have a footer, the br br should be superfluous? I think...

**Review by christianblais — APPROVED** [review]

File: app/javascript/components/layouts/footer.js (hunk 2 above)
Comment (tjoyal) [reply app/javascript/components/layouts/footer.js:7]: I havent dug into why but when missing it complained it was needed and didn't render.

**Pull request merged**